src/compression/mod.rs

```diff
@@ -260,10 +261,32 @@ impl Compression {
     pub fn supports_deep_data(self) -> bool {
         use self::Compression::*;
         match self {
-            Uncompressed | RLE | ZIP1 | ZIP16 => true,
+            Uncompressed | RLE | ZIP1 /* | ZIP16*/ => true,
```

Comment:
Why commenting ZIP16?

Reply:
I have observed that the [C++ code actually does not support ZIP16](https://github.com/AcademySoftwareFoundation/openexr/blob/e46ac0a70cad0d84f8e8be743a9b2d633056da5b/src/lib/OpenEXR/ImfCompressor.cpp#L99-L109) 

(And I couldn't stop myself from fixing right in that moment, it in this PR, because I would otherwise forget to fix it hehe)

Reply:
It's commented because ..eh i don't know... I guess I hoped it might be possible anyways haha. But we can also simply remove the commented ZIP16